### PR TITLE
feat: reduce bundle size

### PR DESCRIPTION
**Pull request by matthprost**
## Type

- Optimisation

#### The following changes where made:

1. Removed 2 `.png` we have in phone number input to keep the best quality one. I also converted it into `.webp` which reduced size even more.

2. Removed `avatar.svg` in `Avatar` component, the image should be imported into the component and not be present in this library. ⚠️ This is might be breaking change as there is no more image as default value

## Relevant logs and/or screenshots

| Page |   Before   |      After |
| :--- | :--------: | ---------: |
| size  | Screenshot 2022-06-09 at 14 42 44 | Screenshot 2022-06-16 at 11 10 48 |

### CODE REVIEW

**Comment by QuiiBz** [comment]
Flags are a bit buggy:

Screenshot 2022-06-10 at 10 13 16

**Comment by matthprost** [comment]
> Flags are a bit buggy:
> 
> Screenshot 2022-06-10 at 10 13 16

Can you tell me your web browser ? On chrome it works fine for me 
Screenshot 2022-06-10 at 17 16 49

**Comment by QuiiBz** [comment]
> 
> Can you tell me your web browser ? On chrome it works fine for me Screenshot 2022-06-10 at 17 16 49

Using Brave, Chrome, Firefox, and Safari, they all have the issue 🤔

Screenshot 2022-06-10 at 21 33 40

File: src/utils/xor.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+type Without<T, U> = { [P in Exclude<keyof T, keyof U>]?: never }
```
Comment (chambo-e) [inline src/utils/xor.ts:1]: into global.d.ts :)

**Review by chambo-e — CHANGES_REQUESTED** [review]

**Review by vincentaudebert — APPROVED** [review]

**Review by QuiiBz — APPROVED** [review]

**Pull request merged**